From a1fc166a0791557a54d91e034dfefd994b11622b Mon Sep 17 00:00:00 2001
From: Phillip Susi <psusi@ubuntu.com>
Date: Sat, 5 Jan 2013 14:59:03 -0500
Subject: [PATCH 56/89] tests: t6003-dm-hide: don't hang on exception

If the parted -l found any exceptions, it would print the prompt, which was
redirected to the log, then hang waiting for input, which never came.  Use
script mode to disable the prompts.
---
 tests/t6003-dm-hide.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/t6003-dm-hide.sh b/tests/t6003-dm-hide.sh
index fce1f31..8618adc 100644
--- a/tests/t6003-dm-hide.sh
+++ b/tests/t6003-dm-hide.sh
@@ -47,7 +47,7 @@ echo 0 2048 linear $d1 0 | dmsetup create $linear_ || fail=1
 dev=/dev/mapper/$linear_
 
 # No "DMRAID-" UUID prefix, hence the device should not show up.
-parted -l >out 2>&1
+parted -s -l >out 2>&1
 grep "^Disk $dev:" out && fail=1
 
 # Unless we perform both dmsetup-remove *and* losetup -d,
@@ -61,7 +61,7 @@ d1=$(loop_setup_ "$f1") || fail=1
 echo 0 2048 linear $d1 0 | dmsetup create $linear_ -u "DMRAID-fake-$$" || fail=1
 
 # Thus, the device should now show up.
-parted -l >out 2>&1
+parted -s -l >out 2>&1
 grep "^Disk $dev:" out || fail=1
 
 Exit $fail
-- 
1.8.5.3

